clients/drcachesim/drcachesim.dox.in

```diff
@@ -721,6 +721,14 @@ running with "-data_prefetcher none" (see \ref sec_drcachesim_ops).
 While misses from software prefetches are included in cache miss files,
 misses from hardware prefetches are not.
 
+The cache simulator can be used to analyze the stream of L2 cache miss
```

Comment:
s/L2/LLC/
(right?  L2 for default, but if someone uses a config file for multi-level it would be LLC?)